xml/System.Text.Json/JsonSerializer.xml

```diff
@@ -269,11 +293,11 @@ The <xref:System.Text.Json.JsonReaderOptions> used to create the instance of the
           
 -or-
 
-<typeparamref name="TValue" /> is not compatible with the JSON.
+<typeparamref name="returnType" /> is not compatible with the JSON.
 
 -or-
 
-There is remaining data in the stream.</exception>
```

Comment:
If there is data left in the Utf8JsonReader, this overload doesn't throw. And using stream was incorrect here.